src/client/Table.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import data from './data.json';
+
+function TableItem(props){
+
+  if(props.seq % 2){
+    return (
+        <tr className="Row-odd">
+            <td>{props.seq}</td>
+            <td>{props.Status}</td>
+            <td>{props.Category}</td>
+            <td>{props.Title}</td>
+            <td>{props.Owner}</td>
+            <td>{props.Priority}</td>
+        </tr>
+    );
+  }
+  else{
+    return (
+      <tr className="Row-even" style={{backgroundColor: 'lightgrey'}}>
```

Comment:
As we can see the difference between `odd` and `even` `table item` is `className` and `style`, you can try to just pass different props for them.

Ex.

```js
function TableItem(props) {
  const isEven = props.seq % 2 // It's better to write the condition like `props.seq % 2 === 0`
  const className = isEven ? 'Row-odd' : 'Row-even'

  return (
    <tr className={className>
      ...
    </tr>
  )
}
```